Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Generator.cs

```diff
@@ -127,6 +127,9 @@ public void Execute(GeneratorExecutionContext context)
                 {
                     annotationReport.CloudFormationTemplatePath = templateHandler.FindTemplate(receiver.ProjectDirectory);
                     annotationReport.ProjectRootDirectory = receiver.ProjectDirectory;
+                    annotationReport.IsCloudFormationDescriptionModificationSuppressed = 
```

Comment:
nit: I'm fine with the implementation, though I'd prefer if naming referred to the purpose instead of implementation description. So instead of `IsCloudFormationDescriptionModificationSuppressed`, I'd prefer if it was `IsTelemetrySuppressed` or something like that. Same applies to `ProjectFileHandler` function `IsCloudFormationDescriptionModificationSuppressed`.

Reply:
Went with `AWSSuppressLambdaAnnotationsTelemetry` after internal discussion. Fixed in amended commit.